pkg/kubelet/kuberuntime/helpers_test.go

```diff
@@ -39,6 +39,64 @@ func seccompLocalhostPath(profileName string) string {
 	return "localhost/" + seccompLocalhostRef(profileName)
 }
 
+func TestIsInitContainerFailed(t *testing.T) {
+	tests := []struct {
+		status      *kubecontainer.Status
+		isFailed    bool
+		description string
+	}{
+		{
+			status: &kubecontainer.Status{
+				State:    kubecontainer.ContainerStateExited,
+				ExitCode: 1,
+			},
+			isFailed:    true,
+			description: "Init container which state is exited and exitcode is non-zero shoud return true",
+		},
+		{
+			status: &kubecontainer.Status{
+				State: kubecontainer.ContainerStateUnknown,
+			},
+			isFailed:    true,
+			description: "Init container which state is unknown should return true",
+		},
+		{
+			status: &kubecontainer.Status{
+				Reason:   "OOMKilled",
+				ExitCode: 0,
+			},
+			isFailed:    true,
+			description: "Init container which state is exited and exitcode is zero shoud return true",
```

Comment:
this description maybe confusing with the line 77,  so can we make it clear for OOMKilled reason?

Reply:
ok. I will see it.